test/091_lido_wrapperunwrapper.ts

```diff
@@ -0,0 +1,82 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address'
+import { constants, id } from '@yield-protocol/utils-v2'
+const { WAD } = constants
+import { ETH, DAI, USDC, WSTETH, STETH } from '../src/constants'
+
+import { ethers, waffle } from 'hardhat'
+import { expect } from 'chai'
+const { deployContract } = waffle
+
+import { STETHMock } from '../typechain/STETHMock'
+import { WstETHMock } from '../typechain/WSTETHMock'
```

Comment:
If the tests pass for you locally, try removing the `artifacts`, `typechain` and `cache` folders, and then doing `yarn build` and `yarn test` again to replicate this error. I can also try.

Reply:
I see the mistake now. Have reverted to using ERC20Mock for steth.